src/reactivity/ref.ts

```diff
@@ -184,7 +184,9 @@ export function shallowRef(raw?: unknown) {
 export function triggerRef(value: any) {
   if (!isRef(value)) return
 
-  value.value = value.value
+  const v = value.value
+  value.value = !v
+  value.value = v
```

Comment:
might need some advice here... this is not ideal for `sync: true` watchers, but I can't call `ob.notify()` without making bigger changes as the reference is not retrievable from `value`